apps/generic-issuance-client/src/helpers/Mutations.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  GenericIssuanceDeletePipelineResponse,
+  GenericIssuanceUpsertPipelineResponse,
+  PipelineDefinition,
+  requestGenericIssuanceDeletePipeline,
+  requestGenericIssuanceUpsertPipeline
+} from "@pcd/passport-interface";
+import { ZUPASS_SERVER_URL } from "../constants";
+
+export const savePipeline = async (
+  userJWT: string,
+  pipelineString: string
+): Promise<GenericIssuanceUpsertPipelineResponse> => {
+  let pipeline: PipelineDefinition;
+  try {
+    pipeline = JSON.parse(pipelineString);
+  } catch (e) {
+    throw new Error(`Invalid JSON object: ${e}`);
+  }
+
+  const saveResponse = await requestGenericIssuanceUpsertPipeline(
+    ZUPASS_SERVER_URL,
+    {
+      jwt: userJWT ?? "",
```

Comment:
i wonder if it's better to assert userJWT exists on the frontend for these auth'd routes

Reply:
fixed